.github/workflows/ci_build_ios.yml

```
@@ -30,7 +30,6 @@ jobs:
         with:
           rust: true
           rust-target: ${{ matrix.target }}
```

🔧 here we still need the token because we use the old setup script
```suggestion
          rust-target: ${{ matrix.target }}
          github_token: ${{ secrets.GITHUB_TOKEN }}
```